tests/test_utils/test_action_utils.py

```diff
@@ -57,3 +63,11 @@ def test_call_agent(self):
         with self.server.register_modules():
             assert execute_agent({}, [], test_node_idtf)[1]
         self.server.remove_modules(module)
+
+    def test_call_action(self):
+        module = ScModuleTest()
+        self.server.add_modules(module)
+        with self.server.register_modules():
+            action_node = prepare_action({}, [])
+            assert execute_action(action_node, test_node_idtf)
+        self.server.remove_modules(module)
```

Comment:
What about testing another functions?

Reply:
Execute action has call action inside, but I think it needs to be added

Reply:
Done